winspool/cairo.go

```diff
@@ -1,3 +1,5 @@
+// +build linux darwin
```

Comment:
I figured out why you did this, and committed the proper fix to the master branch. You can revert everything in winspool now.